etcdctl/go.sum

```diff
@@ -341,8 +341,8 @@ golang.org/x/crypto v0.0.0-20190510104115-cbcb75029529/go.mod h1:yigFU9vqHzYiE8U
 golang.org/x/crypto v0.0.0-20190605123033-f99c8df09eb5/go.mod h1:yigFU9vqHzYiE8UmvKecakEJjdnWj3jj499lnFckfCI=
```

Comment:
Why didn't you update etcdctl/go.mod?

Reply:
This is for the `release-3.5` branch, the dependency is not listed in `etcdctl/go.mod`.

Reply:
Got it, thanks for the clarification.

I see some go.sum files also contain old crypto versions, such as [api/go.sum#L57](https://github.com/etcd-io/etcd/blob/release-3.5/api/go.sum#L57).   Can you fix them?

Reply:
I had run `go mod tidy` in the `api` module, but this produced no changes at all. If I remove the references to the crypto dependency manually from `go.sum`, then running `go mod tidy` brings them back in.

Reply:
They might be dependencies' dependency.

Reply:
Yes, I've looked at the `api` module dependency graph, looks like no top-level dependency bump will cause those transient dependencies to be upgraded the way we want. Some are already at their latest versions, like `protobuf`.

Reply:
@ahrtr, is there still something that needs to be done on my part?

Reply:
Ideally it would be better if you can fix all the old crypto versions, especially which are only in go.sum files.  It may need some effort to figure out what's the exact dependencies which causes the `golang.org/x/crypto` being added into go.sum files. 

It's OK to do it in a separate PR(s) or raise a ticket to track it.

Reply:
I think that `go.sum` is used only for persisting fingerprints for upstream dependencies resolution. 
It doesn't mean we would be depending on the version in any way.

Reply:
```
$ cd etcdctl
$go get golang.org/x/crypto@v0.0.0-20220131195533-30dcbda58838
```

Forces the dependency to be upgraded, even if its a transient dependency. 

go.mod gets modified by: 

```
golang.org/x/crypto v0.0.0-20220131195533-30dcbda58838 // indirect
```
and go.sum as well. 

`go mod tidy` is not removing that change.